eqc/multisig_eqc.erl

```diff
@@ -0,0 +1,67 @@
+-module(multisig_eqc).
+
+-include_lib("eqc/include/eqc.hrl").
+
+-export([prop_multipubkey_test/0]).
+
+-define(HASHTYPE, sha256).
+
+prop_multipubkey_test() ->
+    ?FORALL(
+        {KeyType, {M, N}, Msg},
```

Comment:
On the second look - this doesn't actually get us what we wanted - heterogeneous key combination within a single iteration. Here, just like in the manually-enumerated tests, a single iteration will use a single key type.